applications/FluidDynamicsApplication/tests/test_fluid_rve.py

```diff
@@ -0,0 +1,96 @@
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication as FluidDynamicsApplication # Might be needed for testing
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+import KratosMultiphysics.kratos_utilities as kratos_utilities # Might be needed for testing
+
+from KratosMultiphysics.FluidDynamicsApplication.fluid_rve_analysis import FluidRVEAnalysis
+
+# Previous line will import the Fluid RVE analysis class
+
+class TestFluidRVETest(KratosUnittest.TestCase):
+    
+    def test_fluid_rve_computation_2d(self):
+        #Within location context:
+        with KratosUnittest.WorkFolderScope(".",__file__):
+            with open("FluidRVETest/fluid_rve_test_parameters.json", 'r') as parameter_file:
+                parameters =  KratosMultiphysics.Parameters(parameter_file.read())
+                
+            parameters["solver_settings"]["domain_size"].SetInt(2)
+            parameters["solver_settings"]["model_import_settings"]["input_filename"].SetString("FluidRVETest/fluid_rve_test_2D")
+            parameters["solver_settings"]["material_import_settings"]["materials_filename"].SetString("FluidRVETest/fluid_rve_test_materials_2D.json")
+            parameters["rve_settings"]["boundary_mp_name"].SetString("FluidModelPart.Slip2D.Boundaries")
+            parameters["solver_settings"]["skin_parts"][0].SetString("Slip2D")
+            parameters["processes"]["initial_conditions_process_list"][0]["Parameters"]["value"][0].SetInt(0)
+            parameters["processes"]["initial_conditions_process_list"][0]["Parameters"]["value"][1].SetInt(0)
```

Comment:
Both this and  below were done like this because I did not know if some sort of field was needed. At some point it had a SetString() method with the corresponding "var * G" fields.

Should I delete it regardless?